### PR TITLE
line graph: do not use global and change method names.

### PR DESCRIPTION
**Pull request by artur79**

### CODE REVIEW

File: app/views/teams/dashboard.html.haml [hunk 1]
```diff
@@ -9,4 +9,4 @@
   = render 'summary', presenter: @presenter.team_stats_presenter
 
 :javascript
-  var lineChartData = #{raw @presenter.to_json(graphs: [{ type: :line_chart, data: :team_data }])};
+  lineChartData = #{raw @presenter.to_json(graphs: [{ type: :line_chart, data: :team_data }])};
```
Comment (barbarawuk) [inline app/views/teams/dashboard.html.haml:12]: How come we don't need to set the units and labels here?

**Review by barbarawuk — APPROVED** [review]
Charts all working now, thanks Artur 👍

File: app/views/teams/dashboard.html.haml (hunk 1 above)
Comment (artur79) [reply app/views/teams/dashboard.html.haml:12]: @barbarawuk in presenter initializer

https://github.com/sardjv/murfin_method/blob/master/app/controllers/teams_controller.rb#L33

we use those methods with default values

https://github.com/sardjv/murfin_method/blob/master/app/controllers/concerns/remember_params.rb#L12

Comment (artur79) [reply app/views/teams/dashboard.html.haml:12]: it takes from cookie or from params if not found default one

**Pull request merged**